lib/samson/syslog_formatters.rb

```diff
@@ -4,11 +4,25 @@
 module Samson
   class SyslogFormatter < Syslog::Logger::Formatter
     def call(severity, timestamp, _progname, message)
+      message_h =
+        begin
```

Comment:
raising on every non-hash/json passed in is wasteful ... also code is kinda convoluted
better to dumb it down to this which not as dry but easier to read I hope:

```Ruby
if message.is_a?(Hash)
  message
elsif message.is_a?(String) && message.start_with?('{')
  begin
    JSON.parse(message)
  rescue JSON::ParserError
    {message: message}
  end
else
  {message: message}
end

```

Reply:
old: raise on any non-json string + on any non-string-and-non-hash
new: raise when a string looks like json but is not parseable

Reply:
fine, altered!!